build.gradle

```diff
@@ -2,7 +2,7 @@ ext {
     coroutinesVersion = '1.5.2'
     wordPressUtilsVersion = '2.3.0'
     wordPressLoginVersion = '0.0.8'
-    gutenbergMobileVersion = 'v1.71.0'
+    gutenbergMobileVersion = 'v1.71.1'
```

Comment:
Go home GitHub, [you're… `trunk`](https://github.com/wordpress-mobile/WordPress-Android/pull/15948#issuecomment-1038986666) 🥁 😆   _(…sorry.)_